node/throttled_vm.go

```diff
@@ -20,27 +20,30 @@ func NewThrottledVM(res vm.VM, concurrenyBudget uint, maxQueueLen int32) *Thrott
 }
 
 func (tvm *ThrottledVM) Call(contractAddr, classHash, selector *felt.Felt, calldata []felt.Felt, blockNumber,
-	blockTimestamp uint64, state core.StateReader, network *utils.Network, maxSteps uint64,
+	blockTimestamp uint64, blockVersion string, blockHash *felt.Felt, state core.StateReader,
+	network *utils.Network,
```

Comment:
`maxSteps` was unintentionally removed I believe.

Reply:
Nope, it was intentional. max steps is now in the versioned_constants file (invoke_tx_max_n_steps, validate_max_n_steps)

Reply:
We still need to be able to override that configuration tho.

Reply:
Cool, I'll re-add it then

Reply:
max_steps overrides invoke_tx_max_n_steps